codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2LocalFlowController.java

```diff
@@ -133,8 +140,15 @@ public int initialWindowSize(Http2Stream stream) {
         return state(stream).initialWindowSize();
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * This method must be called from the same event loop context as the {@link ChannelHandlerContext}
```

Comment:
@Scottmitch I wonder if we should just document the interfaces to spell out that they should be called in the event loop.  WDYT?